app/views/partners/children/show.html.erb

```diff
@@ -88,10 +88,10 @@
                         <tr>
                           <td><b><%= child_item_request.created_at.to_date.iso8601 %></b>
                           </th>
-                          <td><%= fetch_valid_item(child_item_request.ordered_item_diaperid)&.dig('name')&.to_s %></th>
+                          <td><%= fetch_valid_item_name(child_item_request.ordered_item_diaperid) %></th>
                           <td><b>
                             <% if (diaperid = child_item_request.picked_up_item_diaperid) && child_item_request.picked_up? %>
-                              <%= valid_items.dig(diaperid, 'name') %>
```

Comment:
`valid_itemsdig(diaperid, 'name')` was a bug! Instead of finding the valid item whose id matches `diaperid`, it would get the `diaperid`-th element in the array.